build.proj

```diff
@@ -79,6 +79,10 @@
     <TestFile Include="$(MSBuildThisFileDirectory)/test/data/external/winequality-white.csv"
       Url="https://archive.ics.uci.edu/ml/machine-learning-databases/wine-quality/winequality-white.csv"
       DestinationFile="$(MSBuildThisFileDirectory)test/data/external/winequality-white.csv" />
+    
+    <TestFile Condition="'$(IncludeBenchmarkData)' == 'true'" Include="$(MSBuildThisFileDirectory)/test/data/external/WikiDetoxAnnotated160kRows.tsv"
+      Url="http://aka.ms/tlc-resources/benchmarks/WikiDetoxAnnotated160kRows.tsv"
```

Comment:
This link doesn't work.  it 404s for me.

Reply:
They removed the dataset until they have the proper place to put the clea approvals